src/commandrunner.cpp

```diff
@@ -270,6 +270,13 @@ void CommandRunner::requestClusters()
     executeMinikubeCommand(args);
 }
 
+void CommandRunner::requestServiceList()
```

Comment:
This should accept a list of args (for example profile) to future proof for the advanced view command.

Reply:
done